Makefile

```diff
@@ -66,9 +66,12 @@ test: go-test go-dependencies-test
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 .PHONY: run
-run: go-generate certmanager $(TMPDIR)k8s-webhook-server/serving-certs/tls.crt
+run: go-generate vendor certmanager $(TMPDIR)k8s-webhook-server/serving-certs/tls.crt
```

Comment:
Do you want to re-introduce the vendor folder?

Reply:
ok, will remove it